hivemind/averaging/averager.py

```diff
@@ -681,15 +697,19 @@ def _background_thread_fetch_current_state(
             get_current_state = get_current_state_ref()
             if get_current_state is None:
                 break
-            state_metadata, state_tensors = get_current_state()
+            state = get_current_state()
+            assert 0 < len(state) <= 3
+            if len(state) != 3:
+                state = tuple(state[:3] + (None,) * (3 - len(state)))
```

Comment:
nit: This makes it a bit clearer.

```suggestion
                state = tuple(state + (None,) * (3 - len(state)))
```